docs/embed-api.rst

```diff
@@ -0,0 +1,122 @@
+Embed API
+=========
+
+Read the Docs allows you to embed content from any of the projects we host.
+This allows reuse of content across sites, making sure the content is always up to date.
+
+.. contents::
+   :local:
+
+Usage
+-----
+
+Check our :doc:`/guides/embedding-content` guide to learn how to use the embed API in your projects.
```

Comment:
I was tempted to move the content from this guide to this section...

Reply:
I think it's fine to have this split. This page is the "raw" documentation of the API's endpoint and the other is a guide that shows some examples.